test-apps/display-test-app/src/frontend/App.ts

```diff
@@ -273,7 +274,9 @@ export class DisplayTestApp {
       // It makes debugging with "pause on caught exceptions" infuriating.
       // ###TODO fix that in the client and remove this
       if (!configuration.noElectronAuth)
-        opts.iModelApp!.authorizationClient = new ElectronRendererAuthorization();
+        opts.iModelApp!.authorizationClient = new ElectronRendererAuthorization({
+          clientId: getConfigurationString("oidcClientId") ?? "imodeljs-spa-test",
```

Comment:
`dta signin` is broken for me and a few others. 
we cant use a client for a spa for a native (electron) app

how does `getConfigurationString` correctly obtain the client id for electron?

fyi @hl662

Reply:
getConfigurationString is an object, and we use [getConfig()](https://github.com/iTwin/itwinjs-core/blob/e579c6d3311c2bd5673a248222bcc13727834d66/test-apps/display-test-app/src/common/DtaConfiguration.ts#L235) in the `common` folder of DTA to assign a value to the client Id. Will open a new PR to make some improvements to the default client id and the getConfig() to differentiate between when to use an electron client id and when to use a browser client id.